docs/source/llms/prompt-engineering/index.rst

```diff
@@ -248,11 +248,11 @@ as follows:
 
 .. _quickstart-score:
 
-Step 12: Score or deploy the best configuration programmatically
```

Comment:
Removed `deploy the best configuration` as the section doesn't really deal with it (though the relevant section is linked below). I can add it back if needed!